build.gradle

```diff
@@ -21,6 +21,7 @@ buildscript {
         classpath 'de.undercouch:gradle-download-task:5.2.1'
         classpath 'io.spring.javaformat:spring-javaformat-gradle-plugin:0.0.38'
         classpath 'com.diffplug.spotless:spotless-plugin-gradle:6.15.0'
+        classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:6.3.1'
```

Comment:
It looks like 6.4.0 has been released since this PR was opened. Any issue upgrading to the latest?